app/js/components/homePage/manageApps/AddAddon.jsx

```diff
@@ -9,41 +9,47 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-export default function AddAddon({ handleClear, handleUpload }) {
+export default function AddAddon({ handleClear, handleUpload, displayManageOwaButtons, displayManageOwaButtonsState }) {
```

Comment:
I think this is line is longer than 80 as specified in the guide, is this not important?